src/format/BitwardenReader.cpp

```diff
@@ -0,0 +1,312 @@
+                auto url = urlObj.toMap().value("uri").toString();
+                if (entry->url().isEmpty()) {
+                    // First url encountered is set as the primary url
+                    entry->setUrl(url);
+                } else {
+                    // Subsequent urls
+                    entry->attributes()->set(QString("KP2A_URL_%1").arg(i), url);
+                    ++i;
+                }
+            }
+        }
+
+        // Parse identity details if present
+        if (itemMap.contains("identity")) {
+            auto idMap = itemMap.value("identity").toMap();
+
+            // Combine name attributes
+            auto attrs = QStringList({idMap.value("title").toString(),
+                                      idMap.value("firstName").toString(),
+                                      idMap.value("middleName").toString(),
+                                      idMap.value("lastName").toString()});
+            attrs.removeAll("");
+            entry->attributes()->set("identity_name", attrs.join(" "));
+
+            // Combine all the address attributes
+            attrs = QStringList({idMap.value("address1").toString(),
+                                 idMap.value("address2").toString(),
+                                 idMap.value("address3").toString()});
+            attrs.removeAll("");
+            auto address = attrs.join("\n") + "\n" + idMap.value("city").toString() + ", "
+                           + idMap.value("state").toString() + " " + idMap.value("postalCode").toString() + "\n"
+                           + idMap.value("country").toString();
+            entry->attributes()->set("identity_address", address);
+
+            // Add the remaining attributes
+            attrs = QStringList({"company", "email", "phone", "ssn", "passportNumber", "licenseNumber"});
+            auto sensitive = QStringList({"ssn", "passportNumber", "licenseNumber"});
+            for (const auto& attr : attrs) {
+                auto value = idMap.value(attr).toString();
+                if (!value.isEmpty()) {
+                    entry->attributes()->set("identity_" + attr, value, sensitive.contains(attr));
+                }
+            }
+
+            // Set the username or push it into attributes if already set
+            auto username = idMap.value("username").toString();
+            if (!username.isEmpty()) {
+                if (entry->username().isEmpty()) {
+                    entry->setUsername(username);
+                } else {
+                    entry->attributes()->set("identity_username", username);
+                }
+            }
+        }
+
+        // Parse card details if present
+        if (itemMap.contains("card")) {
+            auto cardMap = itemMap.value("card").toMap();
+            auto attrs = QStringList({"cardholderName", "brand", "number", "expMonth", "expYear", "code"});
+            auto sensitive = QStringList({"code"});
+            for (const auto& attr : attrs) {
```

Comment:
```suggestion
            const auto cardMap = itemMap.value("card").toMap();
            const auto attrs = QStringList({"cardholderName", "brand", "number", "expMonth", "expYear", "code"});
            const auto sensitive = QStringList({"code"});
            for (const auto& attr : attrs) {
```
At least attrs must be const to avoid detaching.